app/javascript/update_checkout_quantity.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function () {
```

Comment:
As we are doing development in Rails 7.
So, my suggestion is to use turbo-stream and turbo-frame instead of using ajax